templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -0,0 +1,18 @@
+name: ovs-configuration.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Configures OVS with proper host networking configuration
+  # This service is used to move a physical NIC into OVS and reconfigure OVS to use the host IP
+  Wants=network-online.target
+  After=openvswitch.service network-online.target ignition-firstboot-complete.service
```

Comment:
Drop the `ignition-firstboot-complete.service` here; you want `machine-config-daemon-firstboot.service` now probably.